Automattic/wp-calypso
# Gutenboarding: New Acquire Intent design

This PR implements the new design for the acquire intent step on gutenboarding. It doesn't change any significant functionality, just the look&feel. 

![WordPress com (6)](https://user-images.githubusercontent.com/1554855/76091011-01178e00-5fbd-11ea-9f8e-ba94c0953b21.gif)


How to test 
=========
0. Well, not a lot to test here: Just go to http://calypso.localhost:3000/gutenboarding and see everything is looking right, you can enter a vertical and a site title, and that some of the veritcals (examples: 'football', 'dogs', 'coffee') make the color scheme to change when entered.
